core/src/mindustry/entities/bullet/MassDriverBolt.java

```diff
@@ -47,14 +47,17 @@ public void update(Bulletc b){
 
         DriverBulletData data = (DriverBulletData)b.data();
 
-        //if the target is dead, just keep flying until the bullet explodes
-        if(data.to.dead()){
-            return;
-        }
-
         float baseDst = data.from.dst(data.to);
         float dst1 = b.dst(data.from);
         float dst2 = b.dst(data.to);
+        
+        //if the target is dead, oof upon passing the destination point
```

Comment:
Why should the bullet just disappear when it flies past a mass driver that isn't there anymore? There's nothing to stop it.